### PR TITLE
Functions for automatic test generation 120 graphs / 5 events

### PR DESCRIPTION
**Pull request by allegroCoder**

### CODE REVIEW

**Comment by allegroCoder** [comment]
@snowleopard could you merge this PR and remove the PR #29?

**Comment by allegroCoder** [comment]
Sorry, I found a small bug in the file remotion in Windows, I will give my OK for the merge ASAP

File: Test.hs [hunk 1]
```diff
@@ -9,8 +12,15 @@ import Tuura.Scenco
 testPath :: FilePath
 testPath = "test"
 
-techLibPath :: FilePath
-techLibPath = (testPath </> "90nm.genlib")
+techLib :: Library
+techLib = loadLibrary (testPath </> "90nm.genlib")
+
+-- automatic generation
+nGraphs :: Int
```
Comment (snowleopard) [inline Test.hs]: Let's keep `n` at 50. I think it's interesting to cover not the full space of permutations.

Comment (snowleopard) [inline Test.hs]: Actually, `n = 60` may be even better -- covering exactly half of all permutations :-)

**Comment by allegroCoder** [comment]
I think that now you can safely merge everything. :)

**Comment by snowleopard** [comment]
:+1:

**Pull request merged**